valid - override,extraction valid,URL Variable: extractioncorrect,output,build_num,repository_owner,repository_name,build_id,Was our extraction from the buildlog correct?,"Would you go to the whole build log, scroll to the part describing the actual reason the build failed and copy-paste that part back here? Alternatively you could also describe the actual failure reason with your own words.","If you want, please tell us why our extraction was wrong / this one is correct",
,ja,true,"{'EXIT',{error,{enoent,""/home/travis/build/processone/ejabberd/ebin""}}}",4060,processone,ejabberd,557787572,,,,
,ja,true,"ejabberd_SUITE:init_per_group failed Reason: {test_case_failed,{failed_to_create_sql_tables,pgsql,                       {aborted,                           [{severity,'ERROR'},                            {86,ltlt""ERROR""gtgt},                            {code,ltlt""42501""gtgt},                            {message,ltlt""must be owner of relation mqtt_pub""gtgt},                            {file,ltlt""aclchk.c""gtgt},                            {line,3416},                            {routine,ltlt""aclcheck_error""gtgt}]}}}",4142,processone,ejabberd,566947386,,,,
ja,nein,false,"../xml/erl_parse.xml:72: element name: validity error : Element marker is not declared in name list of possible children       ltnamegtltmarker id=""type-af_binelement""/gtaf_binelement(_)lt/namegt                                                                     ^ ../xml/erl_parse.xml:78: element name: validity error : Element marker is not declared in name list of possible children       ltnamegtltmarker id=""type-af_generator""/gtaf_generator()lt/namegt                                                                  ^ ../xml/erl_parse.xml:85: element name: validity error : Element marker is not declared in name list of possible children       ltnamegtltmarker id=""type-af_remote_function""/gtaf_remote_function()gtlt/namegt                                                                               ^",7849,erlang,otp,553211512,"no, the build failed for another reason","What happened in this case is that I provided a pull request with a single commit that fixed some Erlang file and then, somebody else, from the Erlang/OTP team, went ahead and added an extra, erroneous commit to this PR whose intention was to update the documentation.  As you can see here (https://travis-ci.org/erlang/otp/builds/553211512), most of the builds passed except from the last one which builds the documentation.",You can see the discussion on the relevant PR (https://github.com/erlang/otp/pull/2309).,
,ja,false,"t9502-gitweb-standalone-parse-output.sh ............ [31mDubious, test returned 1 (wstat 256, 0x100)[0m [31mFailed 1/13 subtests",4424,git,git,565980859,"yes, this part of the build log describes why the build failed","The ""real"" cause of the failure was that in ""poisoned gettext"" scenarios the code using standard error texts from called programs got unexpected garbage and so failed. I am not sure this is exposed in the logs directly, but at least the git project has a link in the end of logs, under ""ci/print-test-failures.sh"", to detail the failure and let download the relevant files from test workspace.",,
,ja,true,005+ Error: stat1 do not match with stat2 at key value: 8  006+ Error: stat1 do not match with stat2 at key value: atime  005- bool(true)  007- --- Done ---  008+  Dumping stat array 1...,37627,php,php-src,529942305,,,,
,ja,true,"The locale dependent function fprintf(...) appears to be used: src/bitcoin-cli.cpp:                fprintf(stderr, ""RPC password> ""); src/bitcoin-cli.cpp:                fprintf(stderr, ""Wallet passphrase> "");  Unnecessary locale dependence can cause bugs that are very tricky to isolate and fix. Please avoid using locale dependent functions if possible.  Advice not applicable in this specific case? Add an exception by updating the ignore list in test/lint/lint-locale-dependence.sh",42126,bitcoin,bitcoin,567199364,,,,
,ja,true,"The locale dependent function fprintf(...) appears to be used: src/bitcoin-cli.cpp:                fprintf(stderr, ""RPC password> ""); src/bitcoin-cli.cpp:                fprintf(stderr, ""Wallet passphrase> "");  Unnecessary locale dependence can cause bugs that are very tricky to isolate and fix. Please avoid using locale dependent functions if possible.  Advice not applicable in this specific case? Add an exception by updating the ignore list in test/lint/lint-locale-dependence.sh",42128,bitcoin,bitcoin,567200607,,,,
,ja,true,"The locale dependent function fprintf(...) appears to be used: src/bitcoin-cli.cpp:                fprintf(stderr, ""RPC password> ""); src/bitcoin-cli.cpp:                fprintf(stderr, ""Wallet passphrase> "");  Unnecessary locale dependence can cause bugs that are very tricky to isolate and fix. Please avoid using locale dependent functions if possible.  Advice not applicable in this specific case? Add an exception by updating the ignore list in test/lint/lint-locale-dependence.sh",42129,bitcoin,bitcoin,567200724,,,,
,ja,true,005+ Error: stat1 do not match with stat2 at key value: 8  006+ Error: stat1 do not match with stat2 at key value: atime  005- bool(true)  007- --- Done ---  008+  Dumping stat array 1...,37627,php,php-src,529942305,,,,
,ja,true,** (Mix) mix format failed due to --check-formatted. The following files were not formatted:    * lib/elixir/test/elixir/module/types/infer_test.exs,23720,elixir-lang,elixir,574602754,,,,
,ja,true,"1) test pry no sessions (IEx.ServerTest)      [1m[30mtest/iex/server_test.exs:21[0m      [31mAssertion with == failed[0m      [36mcode:  [0massert IEx.pry() == {:error, :no_iex}      [36mleft:  [0m{:error, [31m:refused[0m}      [36mright: [0m{:error, [32m:no_iex[0m}      [36mstacktrace:[0m        (ex_unit 1.10.0-dev) lib/ex_unit/capture_io.ex:159: ExUnit.CaptureIO.do_capture_io/2        (ex_unit 1.10.0-dev) lib/ex_unit/capture_io.ex:128: ExUnit.CaptureIO.do_capture_io/3        test/iex/server_test.exs:22: (test)",23725,elixir-lang,elixir,574613709,,,,
,ja,true,"** (CompileError) (for doctest at) lib/list.ex:102: undefined function i/1     (elixir 1.10.0-dev) src/elixir_locals.erl:108: :elixir_locals.""-ensure_no_undefined_local/3-lc$^0/1-0-""/2     (elixir 1.10.0-dev) src/elixir_locals.erl:109: anonymous fn/3 in :elixir_locals.ensure_no_undefined_local/3",23683,elixir-lang,elixir,572202401,,,,
,ja,true,"** (CompileError) (for doctest at) lib/list.ex:102: undefined function i/1     (elixir 1.10.0-dev) src/elixir_locals.erl:108: :elixir_locals.""-ensure_no_undefined_local/3-lc$^0/1-0-""/2     (elixir 1.10.0-dev) src/elixir_locals.erl:109: anonymous fn/3 in :elixir_locals.ensure_no_undefined_local/3",23684,elixir-lang,elixir,572202566,,,,
ja,nein,false,"1) test does not log converted errors if response already sent (Phoenix.Endpoint.RenderErrorsTest)      [1m[30mtest/phoenix/endpoint/render_errors_test.exs:146[0m      [31mAssertion with == failed[0m      [36mcode:  [0massert capture_log(fn -> try do               try do                 Plug.Conn.send_resp(conn, 200, ""hello"")                 throw(:hello)               catch                 kind, reason ->                   stack = System.stacktrace()",6972,phoenixframework,phoenix,565197107,"close, but not quite correct","1) test does not log converted errors if response already sent (Phoenix.Endpoint.RenderErrorsTest)      test/phoenix/endpoint/render_errors_test.exs:146      Assertion with == failed      code:  assert capture_log(fn -> try do               try do                 Plug.Conn.send_resp(conn, 200, ""hello"")                 throw(:hello)               catch                 kind, reason ->                   stack = System.stacktrace()                   opts = [view: __MODULE__, accepts: ~w""html""]                   Phoenix.Endpoint.RenderErrors.__catch__(conn, kind, reason, stack, opts)               else                 _ ->                   flunk(""function should have failed"")               end             catch               :throw, :hello ->                 :ok             end end) == """"      left:  ""n21:37:31.096 [warn]  no configuration found for otp_app :phoenix and module Phoenix.Endpoint.EndpointTest.NoConfigEndpointn""      right: """"      stacktrace:        test/phoenix/endpoint/render_errors_test.exs:149: (test)",What you extracted did stop in the middle of the test error details.,
,ja,true,"normalize.css[24m   [2m47:6[22m   [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m48:11[22m  [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m83:6[22m   [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m84:11[22m  [31m[31m✖[31m[39m  Unexpected duplicate      [2mdeclaration-block-no-duplicate-properties[22m             ""text-decoration""                                                     [2m84:11[22m  [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m",354,necolas,normalize.css,443134971,,,,
,ja,true,"1) Check challenges        Check challenges (english)          Search and Replace            Check tests syntax              Check for: code>myReplace(""His name is Tom"", ""Tom"", ""John"") /code> should return ""His name is John"".: [0m[31m     expected [Function] to not throw an error but 'SyntaxError: Unexpected token )' was thrown",46830,freeCodeCamp,freeCodeCamp,566117171,,,,
,ja,true,"normalize.css[24m  [2m84:3[22m  [31m[31m✖[31m[39m  Unexpected duplicate        [2mdeclaration-block-no-duplicate-properties[22m           ""text-decoration""",356,necolas,normalize.css,448205109,,,,
,ja,true,"pathParamRequired(retrofit2.RequestFactoryTest)  Time elapsed: 0.018 sec   FAILURE! org.junit.ComparisonFailure: expected:...ue must not be null.[]""> but was:...ue must not be null.[ (parameter #1)     for method Example.method]""> 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) 	at retrofit2.RequestFactoryTest.pathParamRequired(RequestFactoryTest.java:974) 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)",3705,square,retrofit,482520685,,,,
,ja,true,getWithNullUrl(retrofit2.RequestFactoryTest)  Time elapsed: 0.007 sec   ERROR! java.lang.IllegalArgumentException: @Url parameter is null. (parameter #1)     for method Example.method 	at retrofit2.Utils.methodError(Utils.java:52) 	at retrofit2.Utils.parameterError(Utils.java:57) 	at retrofit2.ParameterHandler$RelativeUrl.apply(ParameterHandler.java:71) 	at retrofit2.RequestFactory.create(RequestFactory.java:108) 	at retrofit2.RequestFactoryTest.buildRequest(RequestFactoryTest.java:2887) 	at retrofit2.RequestFactoryTest.buildRequest(RequestFactoryTest.java:2900) 	at retrofit2.RequestFactoryTest.getWithNullUrl(RequestFactoryTest.java:1457),3704,square,retrofit,482509524,,,,
,ja,true,"ERROR: no certificate subject alternative name matches 	requested host name 'openface-models.storage.cmusatyalab.org'. To connect to openface-models.storage.cmusatyalab.org insecurely, use `--no-check-certificate'. + nn4.small2.v1.t7: Error in wget. ERROR: no certificate subject alternative name matches 	requested host name 'openface-models.storage.cmusatyalab.org'. To connect to openface-models.storage.cmusatyalab.org insecurely, use `--no-check-certificate'. + celeb-classifier.nn4.small2.v1.pkl: Error in wget.",581,cmusatyalab,openface,199281591,,,,
,ja,true,"FAIL: tests.openface_batch_represent_tests.test_batch_represent ---------------------------------------------------------------------- Traceback (most recent call last):   File ""/usr/lib/python2.7/dist-packages/nose/case.py"", line 197, in runTest     self.test(*self.arg)   File ""/root/src/openface/tests/openface_batch_represent_tests.py"", line 50, in test_batch_represent     assert p.returncode == 0 AssertionError:  -------------------- >> begin captured stdout << ---------------------",594,cmusatyalab,openface,225393266,,,,
,ja,true,"ERROR: no certificate subject alternative name matches 	requested host name 'openface-models.storage.cmusatyalab.org'. To connect to openface-models.storage.cmusatyalab.org insecurely, use `--no-check-certificate'.",583,cmusatyalab,openface,199292219,,,,
ja,nein,false,"normalize.css[24m   [2m47:6[22m   [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m48:11[22m  [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m83:6[22m   [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m                                           2 spaces                                                              [2m84:11[22m  [31m[31m✖[31m[39m  Unexpected duplicate      [2mdeclaration-block-no-duplicate-properties[22m             ""text-decoration""                                                     [2m84:11[22m  [31m[31m✖[31m[39m  Expected indentation of   [2mindentation[22m",354,necolas,normalize.css,443134971,"close, but not quite correct","47:6   ✖  Expected indentation of   indentation                                           2 spaces                                                              48:11  ✖  Expected indentation of   indentation                                           2 spaces                                                              83:6   ✖  Expected indentation of   indentation                                           2 spaces                                                              84:11  ✖  Unexpected duplicate      declaration-block-no-duplicate-properties             ""text-decoration""                                                     84:11  ✖  Expected indentation of   indentation                                           2 spaces                                                             238:6   ✖  Expected indentation of   indentation                                           2 spaces                                                             239:11  ✖  Expected indentation of   indentation                                           2 spaces                                                             271:6   ✖  Expected indentation of   indentation                                           2 spaces                                                             272:11  ✖  Expected indentation of   indentation                                           2 spaces",The presumed reason only included the first 5 errors,
,ja,true,"/home/travis/build/freeCodeCamp/freeCodeCamp/curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md: 78: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: ""```""]",46864,freeCodeCamp,freeCodeCamp,566612949,,,,
,ja,true,FAIL: TestBoundedForkedSync64Fast (0.18s)     downloader_test.go:679: failed to synchronise blocks: retrieved hash chain is invalid,19585,ethereum,go-ethereum,566009484,,,,
,ja,true,../vendor/github.com/elastic/gosigar/sigar_darwin.go:12:10: fatal error: 'libproc.h' file not found  #include            ^~~~~~~~~~~  1 error generated.,19594,ethereum,go-ethereum,566824752,,,,
,ja,true,> Links    1. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg   > Dupes    None ✓,1635,dkhamsing,open-source-ios-apps,562475335,,,,
,ja,true,> Links    1. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     2. [L1581] 404 https://github.com/mssun/passforios/raw/master/img/preview.gif   > Dupes    None ✓,1633,dkhamsing,open-source-ios-apps,562461458,,,,
,ja,true,> Links    1. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     2. [L1581] 404 https://github.com/mssun/passforios/raw/master/img/preview.gif   > Dupes    None ✓,1632,dkhamsing,open-source-ios-apps,562461418,,,,
ja,nein,false,> Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  → https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example,1630,dkhamsing,open-source-ios-apps,562459124,"close, but not quite correct",Issues :-( > Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  →' https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example    10. [L1300] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png    11. [L1301] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png    12. [L1302] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png    13. [L1360] 301 https://github.com/vanyaland/News  → https://github.com/ivan-magda/News    14. [L1427] 301 https://github.com/vanyaland/Popular-Movies  → https://github.com/ivan-magda/Popular-Movies    15. [L1428] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movie-detail.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movie-detail.png    16. [L1429] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movies-list.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movies-list.png  > Dupes    None ✓,,
,ja,true,"In file included from vis-lua.h:14:0,                  from main.c:15: vis-subprocess.h:21:29: error: unknown type name 'fd_set'  int vis_process_before_tick(fd_set *);                              ^ vis-subprocess.h:22:30: error: unknown type name 'fd_set'  void vis_process_tick(Vis *, fd_set *);                               ^ In file included from vis-lua.h:14:0,                  from vis-core.h:7,",957,martanne,vis,346413726,,,,
,ja,true,"In file included from vis-lua.h:14:0,                  from main.c:15: vis-subprocess.h:21:29: error: unknown type name 'fd_set'  int vis_process_before_tick(fd_set *);                              ^ vis-subprocess.h:22:30: error: unknown type name 'fd_set'  void vis_process_tick(Vis *, fd_set *);                               ^ In file included from vis-lua.h:14:0,                  from vis-core.h:7,",958,martanne,vis,346425105,,,,
ja,nein,false,> Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  → https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example,1631,dkhamsing,open-source-ios-apps,562459179,"close, but not quite correct",Issues :-( > Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  → https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example    10. [L1300] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png    11. [L1301] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png    12. [L1302] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png    13. [L1360] 301 https://github.com/vanyaland/News  → https://github.com/ivan-magda/News    14. [L1427] 301 https://github.com/vanyaland/Popular-Movies  → https://github.com/ivan-magda/Popular-Movies    15. [L1428] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movie-detail.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movie-detail.png    16. [L1429] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movies-list.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movies-list.png  > Dupes    None ✓,,
ja,nein,false,> Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  → https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example,1634,dkhamsing,open-source-ios-apps,562475166,"close, but not quite correct",> Links    01. [L0191] 301 https://github.com/vanyaland/ToThePenny  → https://github.com/ivan-magda/ToThePenny    02. [L0239] 522 http://www.wtfgamersonly.com/wp-content/uploads/2014/08/gba4ios.jpg     03. [L1170] 301 https://github.com/vanyaland/Tagger  → https://github.com/ivan-magda/Tagger    04. [L1171] 301 https://github.com/vanyaland/Tagger/raw/master/Screenshots/main.png  → https://github.com/ivan-magda/Tagger/raw/master/Screenshots/main.png    05. [L1229] 301 https://github.com/vanyaland/UpcomingMovies  → https://github.com/ivan-magda/UpcomingMovies    06. [L1230] 301 https://github.com/vanyaland/UpcomingMovies/raw/master/Screenshots/movies.png  → https://github.com/ivan-magda/UpcomingMovies/raw/master/Screenshots/movies.png    07. [L1268] 301 https://github.com/vanyaland/Californication  → https://github.com/ivan-magda/Californication    08. [L1269] 301 https://github.com/vanyaland/Californication/raw/master/screenshot.png  → https://github.com/ivan-magda/Californication/raw/master/screenshot.png    09. [L1299] 301 https://github.com/vanyaland/MVVM-Example  → https://github.com/ivan-magda/MVVM-Example    10. [L1300] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-apps-list.png    11. [L1301] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-generic-detail.png    12. [L1302] 301 https://github.com/vanyaland/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png  → https://github.com/ivan-magda/MVVM-Example/blob/master/resources/images/mvvm-example-minion.png    13. [L1360] 301 https://github.com/vanyaland/News  → https://github.com/ivan-magda/News    14. [L1427] 301 https://github.com/vanyaland/Popular-Movies  → https://github.com/ivan-magda/Popular-Movies    15. [L1428] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movie-detail.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movie-detail.png    16. [L1429] 301 https://github.com/vanyaland/Popular-Movies/blob/master/res/iOS-movies-list.png  → https://github.com/ivan-magda/Popular-Movies/blob/master/res/iOS-movies-list.png  > Dupes    None ✓,,
,ja,true,"** (throw) ""Words not in order ""exjprop"" and ""ex_progress""""",2162,h4cc,awesome-elixir,506226859,,,,
,ja,true,"io.reactivex.internal.operators.observable.ObservableRefCountTest > publishNoLeak FAILED     java.lang.AssertionError: 13,928,704 -> 113,907,376         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.observable.ObservableRefCountTest.publishNoLeak(ObservableRefCountTest.java:731)",6026,ReactiveX,RxJava,561008032,,,,
,ja,true,"io.reactivex.internal.operators.observable.ObservableRefCountTest > publishNoLeak FAILED     java.lang.AssertionError: 12,973,696 -> 112,950,472         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.observable.ObservableRefCountTest.publishNoLeak(ObservableRefCountTest.java:733)",6027,ReactiveX,RxJava,561016934,,,,
,ja,true,"io.reactivex.internal.operators.observable.ObservableRefCountAltTest > publishNoLeak FAILED     java.lang.AssertionError: 9,283,448 -> 109,750,368         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.observable.ObservableRefCountAltTest.publishNoLeak(ObservableRefCountAltTest.java:725)",6055,ReactiveX,RxJava,565360975,,,,
,ja,true,"io.reactivex.internal.operators.observable.ObservableRefCountAltTest > publishNoLeak FAILED     java.lang.AssertionError: 9,680,016 -> 109,192,144         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.observable.ObservableRefCountAltTest.publishNoLeak(ObservableRefCountAltTest.java:725)",6056,ReactiveX,RxJava,565366738,,,,
,ja,true,"io.reactivex.internal.operators.observable.ObservableRefCountAltTest > publishNoLeak FAILED     java.lang.AssertionError: 9,782,280 -> 109,128,528         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.observable.ObservableRefCountAltTest.publishNoLeak(ObservableRefCountAltTest.java:725)",6057,ReactiveX,RxJava,565371172,,,,
,ja,true,"io.reactivex.internal.operators.flowable.FlowableRefCountTest > publishNoLeak FAILED     java.lang.AssertionError: 19,199,656 -> 115,794,992         at org.junit.Assert.fail(Assert.java:88)         at org.junit.Assert.assertTrue(Assert.java:41)         at io.reactivex.internal.operators.flowable.FlowableRefCountTest.publishNoLeak(FlowableRefCountTest.java:761)",6042,ReactiveX,RxJava,563267830,,,,
,ja,true,"Package size limit has exceeded by 934 B[39m   Package size: [1m[31m103.91 KB[39m[22m   Size limit:   [1m103 KB[22m [90m  With all dependencies, minified and gzipped",1941,postcss,autoprefixer,462651213,,,,
,ja,true,"Package size limit has exceeded by 49 B[39m   Package size: [1m[31m104.05 KB[39m[22m   Size limit:   [1m104 KB[22m [90m  With all dependencies, minified and gzipped",1968,postcss,autoprefixer,468796607,,,,
,ja,true,"FAIL [39m[22m[27m[0m [2mtest/[22m[1mautoprefixer.test.js[22m ([0m[1m[41m11.3s[49m[22m[0m) [1m[31m  [1m● [1mhacks › supports grid layout[39m[22m      [2mexpect([22m[31mreceived[39m[2m).toEqual([22m[32mexpected[39m[2m)[22m      Expected value to equal:       [32m[""autoprefixer: css input>:3:3: Autoplacement does not work without grid-template-rows property"", ""autoprefixer: css input>:12:3: Autoplacement does not work without grid-template-columns property"", ""autoprefixer: css input>:36:3: Can not prefix grid-column-end (grid-column-start is not found)"", ""autoprefixer: css input>:39:3: Can not impliment grid-gap without grid-tamplate-columns"", ""autoprefixer: css input>:39:3: Can not find grid areas: head, nav, main, foot"", ""autoprefixer: css input>:47:3: Can not impliment grid-gap without grid-tamplate-columns"", ""autoprefixer: css input>:47:3: Can not find grid areas: a"", ""autoprefixer: css input>:55:3: Can not impliment grid-gap without grid-tamplate-columns"", ""autoprefixer: css input>:55:3: Can not find grid areas: b"", ""autoprefixer: css input>:63:3: Can not find grid areas: c"", ""autoprefixer: css input>:71:3: Can not find grid areas: d"", ""autoprefixer: css input>:99:3: grid-column-span is not part of final Grid Layout. Use grid-column."", ""autoprefixer: css input>:100:3: grid-row-span is not part of final Grid Layout. Use grid-row."", ""autoprefixer: css input>:101:3: grid-auto-columns is not supported by IE"", ""autoprefixer: css input>:102:3: grid-auto-rows is not supported by IE"", ""autoprefixer: css input>:104:33: auto-fill value is not supported by IE"", ""autoprefixer: css input>:105:30: auto-fit value is not supported by IE"", ""autoprefixer: css input>:121:3: Please do not use display: contents; if you have grid setting enabled"", ""autoprefixer: css input>:125:3: IE does not support align-items on grid containers. Try using align-self on child elements instead: .warn_ie_align > * { align-self: center }"", ""autoprefixer: css input>:130:3: IE does not support justify-items on grid containers. Try using justify-self on child elements instead: .warn_ie_justify > * { justify-self: center }"", ""autoprefixer: css input>:135:3: IE does not support justify-content on grid containers"", ""autoprefixer: css input>:140:3: IE does not support place-items on grid containers. Try using place-self on child elements instead: .warn_place_items > * { place-self: start end }"", ""autoprefixer: css input>:164:3: grid-auto-flow is not supported by IE""][39m     Received:       [31m[""autoprefixer: css input>:3:3: Autoplacement does not work without grid-template-rows property"", ""autoprefixer: css input>:12:3: Autoplacement does not work without grid-template-columns property"", ""autoprefixer: css input>:36:3: Can not prefix grid-column-end (grid-column-start is not found)"", ""autoprefixer: css input>:39:3: Can not impliment grid-gap without grid-template-columns"", ""autoprefixer: css input>:39:3: Can not find grid areas: head, nav, main, foot"", ""autoprefixer: css input>:47:3: Can not impliment grid-gap without grid-template-columns"", ""autoprefixer: css input>:47:3: Can not find grid areas: a"", ""autoprefixer: css input>:55:3: Can not impliment grid-gap without grid-template-columns"", ""autoprefixer: css input>:55:3: Can not find grid areas: b"", ""autoprefixer: css input>:63:3: Can not find grid areas: c"", ""autoprefixer: css input>:71:3: Can not find grid areas: d"", ""autoprefixer: css input>:99:3: grid-column-span is not part of final Grid Layout. Use grid-column."", ""autoprefixer: css input>:100:3: grid-row-span is not part of final Grid Layout. Use grid-row."", ""autoprefixer: css input>:101:3: grid-auto-columns is not supported by IE"", ""autoprefixer: css input>:102:3: grid-auto-rows is not supported by IE"", ""autoprefixer: css input>:104:33: auto-fill value is not supported by IE"", ""autoprefixer: css input>:105:30: auto-fit value is not supported by IE"", ""autoprefixer: css input>:121:3: Please do not use display: contents; if you have grid setting enabled"", ""autoprefixer: css input>:125:3: IE does not support align-items on grid containers. Try using align-self on child elements instead: .warn_ie_align > * { align-self: center }"", ""autoprefixer: css input>:130:3: IE does not support justify-items on grid containers. Try using justify-self on child elements instead: .warn_ie_justify > * { justify-self: center }"", ""autoprefixer: css input>:135:3: IE does not support justify-content on grid containers"", ""autoprefixer: css input>:140:3: IE does not support place-items on grid containers. Try using place-self on child elements instead: .warn_place_items > * { place-self: start end }"", ""autoprefixer: css input>:164:3: grid-auto-flow is not supported by IE""][39m",1978,postcss,autoprefixer,474577649,,,,
,ja,true,FAIL [39m[22m[27m[0m [2mtest/[22m[1mautoprefixer.test.js[22m   [1m● [22mTest suite failed to run      TypeError: Cannot assign to read only property 'Symbol(Symbol.toStringTag)' of object '#process>'        [2mat _default ([22mnode_modules/jest-util/build/createProcessObject.js[2m:85:34)[22m  [0m[7m[1m[31m FAIL [39m[22m[27m[0m [2mtest/[22m[1mprefixes.test.js[22m   [1m● [22mTest suite failed to run,2018,postcss,autoprefixer,502902139,,,,
,ja,true,"Package size limit has exceeded by 624 B[39m   Size limit:   [1m110 KB[22m   Package size: [31m[1m110.61 KB[22m[39m [90mwith all dependencies, minified and gzipped",2062,postcss,autoprefixer,554480449,,,,
,ja,true,confirms_rejects_SUITE:mixed_dead_alive_queues_reject failed on line 195 Reason: expecting_nack_got_ack,3761,rabbitmq,rabbitmq-server,571824621,,,,
,ja,true,confirms_rejects_SUITE:mixed_dead_alive_queues_reject failed on line 195 Reason: expecting_nack_got_ack,3761,rabbitmq,rabbitmq-server,571824621,,,,
,ja,true,confirms_rejects_SUITE:mixed_dead_alive_queues_reject failed on line 195 Reason: expecting_nack_got_ack,3761,rabbitmq,rabbitmq-server,571824621,,,,
ja,nein,false,-> ReactiveCocoa [31m -> [0mReactiveCocoa (10.0.0)      - WARN  | file patterns: The `public_header_files` pattern did not match any file.      - ERROR | xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.      - NOTE  | xcodebuild:  note: Using new build system      - NOTE  | xcodebuild:  note: Planning build,3959,ReactiveCocoa,ReactiveCocoa,551975915,"close, but not quite correct",- ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Association.swift:134:2: error: use of unresolved identifier '_rac_objc_setAssociatedObject' 767    - NOTE  | xcodebuild:  ObjectiveC.objc_setAssociatedObject:2:13: note: did you mean 'objc_setAssociatedObject'? 768    - NOTE  | xcodebuild:  ObjectiveC.objc_getAssociatedObject:2:13: note: did you mean 'objc_getAssociatedObject'? 769    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:120:24: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 770    - NOTE  | xcodebuild:  ObjectiveC._objc_msgForward:3:13: note: '_objc_msgForward' declared here 771    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:133:48: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 772    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:210:52: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 773    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:220:59: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 774    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/NSObject+Intercepting.swift:221:14: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 775    - ERROR | xcodebuild:  /Users/travis/build/ReactiveCocoa/ReactiveCocoa/ReactiveCocoa/ObjC+RuntimeSubclassing.swift:40:45: error: use of unresolved identifier '_rac_objc_msgForward'; did you mean '_objc_msgForward'? 776,"The part you extracted diagnosed that there was a problem, but didn't include the actual error message. What I pasted in was just below and contains the actual error.",
,ja,true,"- ./_site/patterns/acyclic-visitor/index.html   *  internally linking to ../visitor/README.md, which does not exist (line 1) - ./_site/patterns/half-sync-half-async/index.html   *  External link http://www.cs.wustl.edu/~schmidt/PDF/PLoP-95.pdf failed: 404 No error - ./_site/patterns/intercepting-filter/index.html   *  External link http://www.javagyan.com/tutorials/corej2eepatterns/presentation-tier-patterns failed: 403 No error   *  External link https://struts.apache.org/docs/interceptors.html failed: 404 No error htmlproofer 3.0.6 | Error:  HTML-Proofer found 4 failures!",3531,iluwatar,java-design-patterns,565304449,,,,
,ja,true,"1) AtomApplication command-line interface behavior atom:// URLs with a ""core"" host sends a URI message to the most recently focused non-spec window that owns the open locations:      Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.",15567,atom,atom,544815664,,,,
,ja,true,"FAIL: TestSeparator (0.03s)     repo_test.go:75: expected entry to be in form of `* [link] - description.`, got '* [ops](https://github.com/nanovms/ops) - Unikernel Builder/Orchestrator'",4280,avelino,awesome-go,552235410,,,,
,ja,true,"1) ComposerTestRepositoryComposerRepositoryTest::testCanonicalizeUrl with data set #0 ('https://example.org/path/to/file', '/path/to/file', 'https://example.org') Argument 4 passed to ComposerRepositoryComposerRepository::__construct() must be an instance of ComposerUtilHttpDownloader, none given, called in /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php on line 231 and defined  /home/travis/build/composer/composer/src/Composer/Repository/ComposerRepository.php:71 /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php:231  2) ComposerTestRepositoryComposerRepositoryTest::testCanonicalizeUrl with data set #1 ('https://example.org/canonic_url', 'https://example.org/canonic_url', 'https://should-not-see-me.test') Argument 4 passed to ComposerRepositoryComposerRepository::__construct() must be an instance of ComposerUtilHttpDownloader, none given, called in /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php on line 231 and defined  /home/travis/build/composer/composer/src/Composer/Repository/ComposerRepository.php:71",8421,composer,composer,551282890,,,,
,ja,true,> Links  1. 301 https://warehouse.python.org/ → https://pypi.io/ 2. 301 http://www.joke2k.net/faker/ → http://joke2k.net/faker/ 3. http://www.clips.ua.ac.be/pattern Net::ReadTimeout,154,vinta,awesome-python,125333673,,,,
,ja,true,> Links  1. 301 https://warehouse.python.org/ → https://pypi.io/ 2. 301 http://www.joke2k.net/faker/ → http://joke2k.net/faker/,156,vinta,awesome-python,125541453,,,,
,ja,true,_________________________________ test_readme __________________________________[0m  source_root = PosixPath('/home/travis/build/nvbn/thefuck')  [1m    def test_readme(source_root):[0m [1m        with source_root.joinpath('README.md').open() as f:[0m [1m            readme = f.read()[0m [1m    [0m [1m            bundled = source_root.joinpath('thefuck') [0m [1m                                 .joinpath('rules') [0m,1704,nvbn,thefuck,554824331,,,,
,ja,true,"1) test raises error message about correct usage (Mix.Tasks.IexTest)      [1m[30mtest/mix/tasks/iex_test.exs:6[0m      [31m** (ArgumentError) argument error[0m      [36mstacktrace:[0m        (stdlib 3.9) :ets.next(:ac_tab, {:application_master, :runtime_tools})        (stdlib 3.9) ets.erl:775: :ets.do_filter/5        test/test_helper.exs:52: anonymous fn/0 in MixTest.Case.__ex_unit_setup_0/1        (ex_unit 1.10.0-dev) lib/ex_unit/on_exit_handler.ex:142: ExUnit.OnExitHandler.exec_callback/1        (ex_unit 1.10.0-dev) lib/ex_unit/on_exit_handler.ex:128: ExUnit.OnExitHandler.on_exit_runner_loop/0",23643,elixir-lang,elixir,569217362,,,,
nein,nein,false,beam_ssa.erl:127: import lists:umerge/1 is unused,7829,erlang,otp,552509993,"close, but not quite correct",compile: warnings being treated as errors beam_ssa.erl:127: import lists:umerge/1 is unused,The snippet was correct but it was only a warning. So including that warnings are being treated as errors can be helpful.,
,ja,true,"1) test reloads on every request (Phoenix.CodeReloaderTest)      [1m[30mtest/phoenix/code_reloader_test.exs:39[0m      [31m** (ArgumentError) argument error[0m      [36mcode: [0m:erlang.trace(pid, true, [:receive])      [36mstacktrace:[0m        :erlang.trace(nil, true, [:receive])        test/phoenix/code_reloader_test.exs:41: (test)",6970,phoenixframework,phoenix,565138076,,,,
,ja,true,Z:appdatadistbuild/libHSShellCheck-0.6.0-BXGxxcYEANCBISlPEsuG9L.a(TTY.o):fake:(.text+0x3e85): undefined reference to `ShellCheckzm0zi6zi0zmBXGxxcYEANCBISlPEsuG9L_ShellCheckziFixer_zdwapplyFix_info'  Z:appdatadistbuild/libHSShellCheck-0.6.0-BXGxxcYEANCBISlPEsuG9L.a(TTY.o):fake:(.data.rel.ro+0x2bc): undefined reference to `ShellCheckzm0zi6zi0zmBXGxxcYEANCBISlPEsuG9L_ShellCheckziFixer_zdwapplyFix_closure'  collect2.exe: error: ld returned 1 exit status,374,koalaman,shellcheck,469308814,,,,
,ja,true,"src/ShellCheck/Fixer.hs:38:10: error: [0m[ 6 of 19] Compiling Paths_ShellCheck ( dist/build/autogen/Paths_ShellCheck.hs, dist/build/Paths_ShellCheck.o ) [ 7 of 19] Compiling ShellCheck.Data  ( src/ShellCheck/Data.hs, dist/build/ShellCheck/Data.o ) [91m    Not in scope: type constructor or class `Semigroup'",376,koalaman,shellcheck,471475140,,,,
,ja,true,shellcheck.hs:442:19: error:     Variable not in scope: doesPathExist :: [Char] -> IO Bool,417,koalaman,shellcheck,501296412,,,,
,ja,true,shellcheck.hs:442:19: error:     Variable not in scope: doesPathExist :: [Char] -> IO Bool,418,koalaman,shellcheck,501296929,,,,
,ja,true,shellcheck.hs:31:1: error:     Failed to load interface for 'ShellCheck.Formatter.JSON1'     Locations searched:       ShellCheck/Formatter/JSON1.hs       ShellCheck/Formatter/JSON1.lhs       ShellCheck/Formatter/JSON1.hsig       ShellCheck/Formatter/JSON1.lhsig       src/ShellCheck/Formatter/JSON1.hs       src/ShellCheck/Formatter/JSON1.lhs       src/ShellCheck/Formatter/JSON1.hsig,453,koalaman,shellcheck,540436118,,,,
,ja,true,"src/ShellCheck/Formatter/Diff.hs:183:31: error:[0m[91m     * Variable not in scope:[0m[91m (>) :: v ->[0m[91m v -> v     * Perhaps you meant one of these:[0m[91m [0m[91m        [0m[91m`*' (imported from[0m[91m Prelude), `*>' (imported from Prelude), [0m[91m        [0m[91m`$>' (imported from[0m[91m Prelude)",459,koalaman,shellcheck,549543406,,,,
,ja,true,cabal.exe: Couldn't establish HTTP connection. Possible cause: HTTP proxy  server is down.,465,koalaman,shellcheck,553130518,,,,
,ja,true,"+docker build -t :current . invalid argument "":current"" for t: invalid reference format",483,koalaman,shellcheck,563381753,,,,
,ja,true,"container_linux.go:265: starting container process caused ""exec: ""-c"": executable file not found in $PATH""",479,koalaman,shellcheck,563353767,,,,
nein,nein,false,Unable to find image 'koalaman/shellcheck:current' locally docker: Error response from daemon: manifest for koalaman/shellcheck:current not found.,452,koalaman,shellcheck,540421319,"no, the build failed for another reason","Downloading the latest package list from hackage.haskell.org < socket: 10>: hGetBufSome: resource vanished (Connection reset by peer) The command '/bin/sh -c cabal update && cabal install --dependencies-only --ghc-options=""-optlo-Os -split-sections""' returned a non-zero code: 1",,
,ja,true,"[1;31mUse IO::Interactive::is_interactive() instead of -t at line 69, column 28.  See page 218 of PBP.  (Severity: 5)[0m [1;31mUse IO::Interactive::is_interactive() instead of -t at line 203, column 28.  See page 218 of PBP.  (Severity: 5)",469,major,MySQLTuner-perl,384620013,,,,
nein,nein,false,operator: notOk     expected: false     actual:   1     at: Test. (/home/travis/build/airbnb/javascript/packages/eslint-config-airbnb/test/test-react-order.js:57:7)     stack: |-       Error: no errors           at Test.assert [as _assert] (/home/travis/build/airbnb/javascript/packages/eslint-config-airbnb/node_modules/tape/lib/test.js:225:54)           at Test.bound [as _assert] (/home/travis/build/airbnb/javascript/packages/eslint-config-airbnb/node_modules/tape/lib/test.js:77:32)           at Test.notOK (/home/travis/build/airbnb/javascript/packages/eslint-config-airbnb/node_modules/tape/lib/test.js:357:10)           at Test.bound (/home/travis/build/airbnb/javascript/packages/eslint-config-airbnb/node_modules/tape/lib/test.js:77:32),1942,airbnb,javascript,556125301,"no, the build failed for another reason","This output is part of the tests; it's TAP output (which is trivial to programmatically parse). All the test jobs passed - the *lint* job is what failed.  It's very confusing that you have a tape error, though, when all the tests passed.",,
ja,nein,false,"Could not find artifact clojure-complete:clojure-complete:jar:0.2.4 in central (https://repo1.maven.org/maven2/) Could not transfer artifact clojure-complete:clojure-complete:jar:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version Could not transfer artifact clojure-complete:clojure-complete:pom:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version This could be due to a typo in :dependencies, file system permissions, or network issues. If you are behind a proxy, try setting the 'http_proxy' environment variable. Could not find artifact clojure-complete:clojure-complete:jar:0.2.4 in central (https://repo1.maven.org/maven2/) Could not transfer artifact clojure-complete:clojure-complete:jar:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version Could not transfer artifact clojure-complete:clojure-complete:pom:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version This could be due to a typo in :dependencies, file system permissions, or network issues. If you are behind a proxy, try setting the 'http_proxy' environment variable.",2454,technomancy,leiningen,239201655,"close, but not quite correct","I think your log extraction / identification is ok, but the root cause was actually a problem in the Travis CI environment rather than the build.  It's been a few years so I can't 100% remember the details, however I can see even in the pull request that my local builds of the same code with the same build toolchain were succeeding at that time.",,
ja,nein,false,"Could not find artifact clojure-complete:clojure-complete:jar:0.2.4 in central (https://repo1.maven.org/maven2/) Could not transfer artifact clojure-complete:clojure-complete:jar:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version Could not transfer artifact clojure-complete:clojure-complete:pom:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version This could be due to a typo in :dependencies, file system permissions, or network issues. If you are behind a proxy, try setting the 'http_proxy' environment variable. Could not find artifact clojure-complete:clojure-complete:jar:0.2.4 in central (https://repo1.maven.org/maven2/) Could not transfer artifact clojure-complete:clojure-complete:jar:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version Could not transfer artifact clojure-complete:clojure-complete:pom:0.2.4 from/to clojars (https://repo.clojars.org/): Received fatal alert: protocol_version This could be due to a typo in :dependencies, file system permissions, or network issues. If you are behind a proxy, try setting the 'http_proxy' environment variable.",2456,technomancy,leiningen,239388390,"close, but not quite correct","I think your log extraction / identification is ok, but the root cause was actually a problem in the Travis CI environment rather than the build.  It's been a few years so I can't 100% remember the details, however I can see even in the pull request that my local builds of the same code with the same build toolchain were succeeding at that time.",,
,ja,true,1) ComposerTestDependencyResolverSolverTest::testUnsatisfiableRequires Failed asserting that two strings are equal. --- Expected +++ Actual @@ @@   - It's a private package and you forgot to add a custom repository to find it + - It's a private package and you forgot to configure the required authentication credentials    Read  for further common problems.',8410,composer,composer,547576002,,,,
,ja,true,"1) ComposerTestInstallerTest::testIntegration with data set ""solver-problems.test"" ('solver-problems.test', 'Test the error output of solv...blems.', null, array(array(array('package', array(array('unstable/package', '2.0.0-alpha'), array('unstable/package', '1.0.0'), array('requirer/pkg', '1.0.0', array('1.0.0')), array('dependency/pkg', '2.0.0'), array('dependency/pkg', '1.0.0'), array('stable-requiree-excluded/pkg', '1.0.1'), array('stable-requiree-excluded/pkg', '1.0.0')))), array('2.*', '1.*', '1.*', '2.*', '1.0.1')), array(), array(array('stable-requiree-excluded/pkg', '1.0.0')), 'update unstable/package requi...cy/pkg', array(), 'Loading composer repositories...blems.', 'n', 2) Failed asserting that format description matches text. --- Expected +++ Actual @@ @@   - It's a private package and you forgot to add a custom repository to find it + - It's a private package and you forgot to configure the required authentication credentials    Read  for further common problems.",8411,composer,composer,547593737,,,,
,ja,true,"1) ComposerTestInstallerTest::testIntegration with data set ""solver-problems.test"" ('solver-problems.test', 'Test the error output of solv...blems.', null, array(array(array('package', array(array('unstable/package', '2.0.0-alpha'), array('unstable/package', '1.0.0'), array('requirer/pkg', '1.0.0', array('1.0.0')), array('dependency/pkg', '2.0.0'), array('dependency/pkg', '1.0.0'), array('stable-requiree-excluded/pkg', '1.0.1'), array('stable-requiree-excluded/pkg', '1.0.0')))), array('2.*', '1.*', '1.*', '2.*', '1.0.1')), array(), array(array('stable-requiree-excluded/pkg', '1.0.0')), 'update unstable/package requi...cy/pkg', array(), 'Loading composer repositories...blems.', 'n', 2) Failed asserting that format description matches text. --- Expected +++ Actual @@ @@   - It's a private package and you forgot to add a custom repository to find it + - It's a private package and you forgot to configure the required authentication credentials    Read  for further common problems.",8412,composer,composer,547594407,,,,
,ja,true,"1) ComposerTestRepositoryComposerRepositoryTest::testCanonicalizeUrl with data set #0 ('https://example.org/path/to/file', '/path/to/file', 'https://example.org') Argument 4 passed to ComposerRepositoryComposerRepository::__construct() must be an instance of ComposerUtilHttpDownloader, none given, called in /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php on line 231 and defined  /home/travis/build/composer/composer/src/Composer/Repository/ComposerRepository.php:71 /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php:231  2) ComposerTestRepositoryComposerRepositoryTest::testCanonicalizeUrl with data set #1 ('https://example.org/canonic_url', 'https://example.org/canonic_url', 'https://should-not-see-me.test') Argument 4 passed to ComposerRepositoryComposerRepository::__construct() must be an instance of ComposerUtilHttpDownloader, none given, called in /home/travis/build/composer/composer/tests/Composer/Test/Repository/ComposerRepositoryTest.php on line 231 and defined  /home/travis/build/composer/composer/src/Composer/Repository/ComposerRepository.php:71",8421,composer,composer,551282890,,,,
,ja,true,PHP Fatal error:  Class 'ComposerUtilResponse' not found in /home/travis/build/composer/composer/src/Composer/Util/RemoteFilesystem.php on line 286,8486,composer,composer,566982103,,,,
,ja,true,Error                                                                                                              -----------------------------------------------------------------------------------------------------------------    Ignored error pattern ~^Anonymous function has an unused use $fileName.$~ was not matched in reported errors.,8479,composer,composer,566904372,,,,
,ja,true,1) ComposerTestUtilStreamContextFactoryTest::testHttpsProxyOverride Failed asserting that two arrays are equal. --- Expected +++ Actual @@ @@  Array (      'http' => Array ( -        'proxy' => 'ssl://woopproxy.net:443' +        'proxy' => 'tcp://woopproxy.net:443'          'request_fulluri' => true,8461,composer,composer,565348117,,,,
,ja,true,1) ComposerTestAllFunctionalTest::testIntegration with data set #0 (SymfonyComponentFinderSplFileInfo Object (...)),8494,composer,composer,567101801,,,,
,ja,true,"An error occurred while loading ./spec/requests/users_controller_spec.rb. [31mFailure/Error: [0m[32mreturn[0m load_without_bootsnap(resolved, wrap)[0m [31m[0m [31mSyntaxError:[0m [31m  /home/travis/build/discourse/discourse/spec/requests/users_controller_spec.rb:3520: syntax error, unexpected end-of-input, expecting end[0m [36m# /home/travis/build/discourse/discourse/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'[0m [36m# /home/travis/build/discourse/discourse/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'[0m [36m# /home/travis/build/discourse/discourse/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `block in load'[0m [36m# /home/travis/build/discourse/discourse/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'[0m [36m# /home/travis/build/discourse/discourse/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'[0m",55233,discourse,discourse,566325071,,,,
,ja,true,"configure: error: in `/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-754a2cc36ac04a65/out/build': configure: error: C compiler cannot create executables See `config.log' for more details thread 'main' panicked at 'command did not execute successfully: ""sh"" ""/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-754a2cc36ac04a65/out/jemalloc/configure"" ""--disable-cxx"" ""--with-jemalloc-prefix=_rjem_"" ""--with-private-namespace=_rjem_"" ""--host=i686-unknown-linux-musl"" ""--build=x86_64-unknown-linux-gnu"" ""--prefix=/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-754a2cc36ac04a65/out"" expected success, got: exit code: 77', /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/jemalloc-sys-0.3.0/build.rs:370:9 stack backtrace:    0:     0x5597a5c1a753 - std::sys::unix::backtrace::tracing::imp::unwind_backtrace::h6bfa65e4a0f076ef                                at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39    1:     0x5597a5c15d1b - std::sys_common::backtrace::_print::h1a592fe565d3945f                                at src/libstd/sys_common/backtrace.rs:71",2033,BurntSushi,ripgrep,524197508,,,,
,ja,true,"configure: error: in `/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/build': configure: error: C compiler cannot create executables See `config.log' for more details thread 'main' panicked at 'command did not execute successfully: ""sh"" ""/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/jemalloc/configure"" ""--disable-cxx"" ""--with-jemalloc-prefix=_rjem_"" ""--with-private-namespace=_rjem_"" ""--host=i686-unknown-linux-musl"" ""--build=x86_64-unknown-linux-gnu"" ""--prefix=/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out"" expected success, got: exit code: 77', /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/jemalloc-sys-0.3.0/build.rs:370:9 stack backtrace:    0:     0x55ffe43f9ad3 - std::sys::unix::backtrace::tracing::imp::unwind_backtrace::h805294fd9a8a1853                                at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39    1:     0x55ffe43f509b - std::sys_common::backtrace::_print::h563674885828f2f5                                at src/libstd/sys_common/backtrace.rs:71",2034,BurntSushi,ripgrep,524343295,,,,
,ja,true,note[0m[0m: /target/i686-unknown-linux-musl/debug/deps/libjemalloc_sys-efcf9f5425698221.rlib(arena.pic.o): In function `ffs_llu':[0m [0m          /target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/build/../jemalloc/include/jemalloc/internal/bit_util.h:17: undefined reference to `__ffsdi2'[0m [0m          collect2: error: ld returned 1 exit status,2040,BurntSushi,ripgrep,524440676,,,,
,ja,true,"configure: error: in `/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/build': configure: error: C compiler cannot create executables See `config.log' for more details thread 'main' panicked at 'command did not execute successfully: ""sh"" ""/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/jemalloc/configure"" ""--disable-cxx"" ""--with-jemalloc-prefix=_rjem_"" ""--with-private-namespace=_rjem_"" ""--host=i686-unknown-linux-musl"" ""--build=x86_64-unknown-linux-gnu"" ""--prefix=/home/travis/build/BurntSushi/ripgrep/target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out"" expected success, got: exit code: 77', /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/jemalloc-sys-0.3.0/build.rs:370:9 stack backtrace:    0:     0x5652fb2fcad3 - std::sys::unix::backtrace::tracing::imp::unwind_backtrace::h805294fd9a8a1853                                at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39    1:     0x5652fb2f809b - std::sys_common::backtrace::_print::h563674885828f2f5                                at src/libstd/sys_common/backtrace.rs:71",2039,BurntSushi,ripgrep,524438701,,,,
,ja,true,"thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: ""No such file or directory"" }', src/libcore/result.rs:999:5 note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.",2037,BurntSushi,ripgrep,524433448,,,,
,ja,true,note[0m[0m: /target/i686-unknown-linux-musl/debug/deps/libjemalloc_sys-efcf9f5425698221.rlib(arena.pic.o): In function `ffs_llu':[0m [0m          /target/i686-unknown-linux-musl/debug/build/jemalloc-sys-8599e81d749c60c3/out/build/../jemalloc/include/jemalloc/internal/bit_util.h:17: undefined reference to `__ffsdi2'[0m [0m          collect2: error: ld returned 1 exit status,2038,BurntSushi,ripgrep,524435666,,,,
,ja,true,sbt.ResolveException: unresolved dependency: org.scala-lang#scala-compiler;2.12.9-bin-5d0a905-SNAPSHOT: not found 	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:320) 	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:191) 	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:168) 	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156) 	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156) 	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:133) 	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57) 	at sbt.IvySbt$$anon$4.call(Ivy.scala:65) 	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93),9480,scala,scala,567103817,,,,
,ja,true,!![0m[0m  754 - neg/t742.scala                            [compilation failed] [1m[36m##### Log file '/home/travis/build/scala/scala/test/files/neg/t742-neg.log' from failed test ##### [0m[0m error: head of empty list,9468,scala,scala,566726054,,,,
,ja,true,error[0m] [0m1 of 13 test tasks failed:[0m [0m[[31merror[0m] [0m- partest instrumented presentation[0m [0m[[31merror[0m] [0m  - test/it:testOnly failed: sbt.TestsFailedException: Tests unsuccessful[0m java.lang.RuntimeException 	at $e1fdba7da5812f40a13c$$anonfun$root$27.apply(build.sbt:1031) 	at $e1fdba7da5812f40a13c$$anonfun$root$27.apply(build.sbt:973) 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40) 	at sbt.std.Transform$$anon$4.work(System.scala:63) 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228),9493,scala,scala,567174876,,,,
,ja,true,"error[0m] [0m/home/travis/build/scala/scala/src/reflect/scala/reflect/internal/ReificationSupport.scala:614: ')' expected but 'else' found.[0m [0m[[31merror[0m] [0m        if (sumSize(unlifted.flatten, 0) == sumSize(lst.flatten, 0) Some(unlifted) else None[0m [0m[[31merror[0m] [0m                                                                                   ^[0m [0m[[31merror[0m] [0mone error found[0m [0m[[31merror[0m] [0m(reflect/compile:[31mcompileIncremental[0m) Compilation failed[0m [0m[[31merror[0m] [0mTotal time: 120 s, completed Aug 2, 2019 8:51:01 PM[0m",9484,scala,scala,567119813,,,,
,ja,true,[info] - Requests exceeding limit should increase dropped_request count *** FAILED *** [info]   10.0 was not greater than 10 (ConcurrencyEndToEndTest.scala:73),2849,twitter,finagle,572992106,,,,
,ja,true,[0K$ ./check_integrity.sh $(find bin | cut -b 5- | xargs) grep: Commands.md: No such file or directory Add git-contrib in the list of commands in Commands.md,284,tj,git-extras,540384622,,,,
,ja,true,[0K$ ./check_integrity.sh $(find bin | cut -b 5- | xargs) man/git-line-summary.md is required for bin/git-line-summary,310,tj,git-extras,562612733,,,,
,ja,true,"error[0m] [0m[0m [0m [0m[[0m[31merror[0m] [0m[0m1 of 17 test tasks failed:[0m [0m[[0m[31merror[0m] [0m[0m- library/mimaReportBinaryIssues[0m [0m[[0m[31merror[0m] [0m[0m  - library/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-library: Binary compatibility check failed![0m [0m[[0m[31merror[0m] [0m[0mSaved log of library/mimaReportBinaryIssues:[0m [0m[[0m[31merror[0m] [0m[0m * method processUnicode(java.lang.String)java.lang.String in object scala.StringContext does not have a correspondent in other version[0m [0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectMissingMethodProblem](""scala.StringContext.processUnicode"")[0m [0m[[0m[31merror[0m] [0m[0mtestAll failed due to previous errors[0m [0m[[0m[31merror[0m] [0m[0m([31mtestAll[0m) testAll failed due to previous errors[0m",9474,scala,scala,566852148,,,,
,ja,true,"error[0m] [0m[0m [0m [0m[[0m[31merror[0m] [0m[0m1 of 17 test tasks failed:[0m [0m[[0m[31merror[0m] [0m[0m- library/mimaReportBinaryIssues[0m [0m[[0m[31merror[0m] [0m[0m  - library/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-library: Binary compatibility check failed![0m [0m[[0m[31merror[0m] [0m[0mSaved log of library/mimaReportBinaryIssues:[0m [0m[[0m[31merror[0m] [0m[0m * method processUnicode(java.lang.String)java.lang.String in object scala.StringContext does not have a correspondent in other version[0m [0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectMissingMethodProblem](""scala.StringContext.processUnicode"")[0m [0m[[0m[31merror[0m] [0m[0mtestAll failed due to previous errors[0m [0m[[0m[31merror[0m] [0m[0m([31mtestAll[0m) testAll failed due to previous errors[0m",9475,scala,scala,566866822,,,,
,ja,true,,,,,,,,,
,ja,true,fatal: unable to access 'https://github.com/vim/vim/': Empty reply from server,908,junegunn,vim-plug,445184806,,,,
,ja,true,[0K$ ./check_integrity.sh $(find bin | cut -b 5- | xargs) Skip command rscp Run 'make docs' to create man/git-rename-remote.1 and man/git-rename-remote.html,261,tj,git-extras,525540804,,,,
ja,nein,false,"error[0m] [0m[0m/home/travis/build/akka/akka/akka-stream/src/main/scala/akka/stream/impl/ReactiveStreamsCompliance.scala:133:47: The unicode arrow `⇒` is deprecated, use `=>` instead. If you still wish to display it as one character, consider using a font with programming ligatures such as Fira Code.[0m [0m[[0m[31merror[0m] [0m[0m      case s: SubscriptionWithCancelException ⇒ s.cancel(cause)[0m [0m[[0m[31merror[0m] [0m[0m                                              ^[0m [0m[[0m[31merror[0m] [0m[0m/home/travis/build/akka/akka/akka-stream/src/main/scala/akka/stream/impl/ReactiveStreamsCompliance.scala:134:14: The unicode arrow `⇒` is deprecated, use `=>` instead. If you still wish to display it as one character, consider using a font with programming ligatures such as Fira Code.[0m [0m[[0m[31merror[0m] [0m[0m      case s ⇒ s.cancel()[0m [0m[[0m[31merror[0m] [0m[0m             ^[0m [0m[[0m[31merror[0m] [0m[0m/home/travis/build/akka/akka/akka-stream/src/main/scala/akka/stream/impl/ReactiveStreamsCompliance.scala:136:24: The unicode arrow `⇒` is deprecated, use `=>` instead. If you still wish to display it as one character, consider using a font with programming ligatures such as Fira Code.[0m [0m[[0m[31merror[0m] [0m[0m      case NonFatal(t) ⇒ throw new SignalThrewException(""It is illegal to throw exceptions from cancel(), rule 3.15"", t)[0m [0m[[0m[31merror[0m] [0m[0m                       ^[0m [0m[[0m[31merror[0m] [0m[0m/home/travis/build/akka/akka/akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala:179:24: The unicode arrow `⇒` is deprecated, use `=>` instead. If you still wish to display it as one character, consider using a font with programming ligatures such as Fira Code.[0m",6442,akka,akka,562556292,"close, but not quite correct",It's basically correct but it truncated a few lines of the full error at the end.   Btw. filtering out or actually interpreting ANSI sequences would help a lot with readability of those traces ;),,
,ja,true,"The command ""test/run !"" exited with 1.",885,junegunn,vim-plug,398587466,,,,
,ja,true,"The command ""test/run !"" exited with 1.",885,junegunn,vim-plug,398587466,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7ffc53357450 T0) = ==16563==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34125,neovim,neovim,567659224,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7ffe476412b0 T0) = ==16744==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34130,neovim,neovim,567681636,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7fffab6f2bd0 T0) = ==16550==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34131,neovim,neovim,567681970,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7ffc1654d3f0 T0) = ==16456==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34134,neovim,neovim,567686448,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7ffe628bb830 T0) = ==16389==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34135,neovim,neovim,567689244,,,,
,ja,true,app/assets/javascripts/discourse/routes/about.js.es6,55210,discourse,discourse,565586706,,,,
ja,nein,false,FAIL: TestAlpha/Testing_Frameworks (0.00s)         repo_test.go:131: expected 'go-carpet' but actual is 'gobdd'         repo_test.go:131: expected 'go-cmp' but actual is 'go-carpet'         repo_test.go:131: expected 'go-mutesting' but actual is 'go-cmp'         repo_test.go:131: expected 'go-testdeep' but actual is 'go-mutesting'         repo_test.go:131: expected 'go-vcr' but actual is 'go-testdeep'         repo_test.go:131: expected 'gobdd' but actual is 'go-vcr'         repo_test.go:135: expected order is:             assert             badio,4376,avelino,awesome-go,566068072,"close, but not quite correct",I think that the order of libraries is incorrect.,,
,ja,false,app/assets/javascripts/discourse/routes/about.js.es6,55210,discourse,discourse,565586706,"yes, this part of the build log describes why the build failed",,"We use a code formatter for our JS files called Prettier, and linter called Eslint (also Rubocop for ruby). The reason you extracted gave me the correct file that failed, which is really good, but I think if it could tell me which tool (eslint vs prettier) the file failed, that would be amazing!  Nice work and keep it up!",
,ja,true,"[16:40:38] E/launcher - Error: TSError: ⨯ Unable to compile TypeScript e2e/app.e2e-spec.ts (5,1): Cannot find name 'describe'. (2304) e2e/app.e2e-spec.ts (7,13): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (11,57): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (15,42): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (45,9): Cannot find name 'expect'. (2304) e2e/app.e2e-spec.ts (47,9): Cannot find name 'expect'. (2304)",55571,angular,angular,499899891,,,,
,ja,true,"[17:11:01] E/launcher - Error: TSError: ⨯ Unable to compile TypeScript e2e/app.e2e-spec.ts (5,1): Cannot find name 'describe'. (2304) e2e/app.e2e-spec.ts (7,13): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (11,57): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (15,42): Argument of type '() => void' is not assignable to parameter of type '(done: any) => Promisevoid>'.   Type 'void' is not assignable to type 'Promisevoid>'. (2345) e2e/app.e2e-spec.ts (45,9): Cannot find name 'expect'. (2304) e2e/app.e2e-spec.ts (47,9): Cannot find name 'expect'. (2304)",55576,angular,angular,502664562,,,,
,ja,true,"1) NgPackagesInstaller installLocalDependencies() when there is no local package marker should overwrite package.json with modified config   Message: [31m    Expected spy writeFileSync to have been called with [ '/home/travis/build/angular/angular/aio/root/dir/package.json', '{       ""dependencies"": {         ""@angular/core"": ""file:/home/travis/build/angular/angular/dist/packages-dist/core"",         ""@angular/common"": ""file:/home/travis/build/angular/angular/dist/packages-dist/common"",         ""rxjs"": ""^6.4.0""       },       ""devDependencies"": {         ""@angular/compiler-cli"": ""file:/home/travis/build/angular/angular/dist/tools/@angular/compiler-cli"",",55578,angular,angular,502677197,,,,
nein,nein,false,Error: Cannot find module './db.json'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/mime-types/node_modules/mime-db/index.js:11:18)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13456,electron,electron,303140404,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 52.84.14.226 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
nein,nein,false,Error: Cannot find module 'fast-deep-equal'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/har-validator/node_modules/ajv/lib/compile/resolve.js:4:13)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13457,electron,electron,303144317,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 13.33.164.241 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
nein,nein,false,Error: Cannot find module 'fast-deep-equal'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/har-validator/node_modules/ajv/lib/compile/resolve.js:4:13)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13458,electron,electron,303153628,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 13.33.164.32 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
nein,nein,false,Error: Cannot find module 'fast-deep-equal'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/har-validator/node_modules/ajv/lib/compile/resolve.js:4:13)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13459,electron,electron,303172126,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 52.84.14.76 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
nein,nein,false,Error: Cannot find module 'fast-deep-equal'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/har-validator/node_modules/ajv/lib/compile/resolve.js:4:13)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13462,electron,electron,303184368,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 52.84.14.228 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
,ja,true,"> Links    01. [L0410] 301 https://github.com/CoreCharts/CoreCharts  → https://github.com/cagricolak/CoreCharts    02. [L0500] 301 https://github.com/arikis/Overdrive  → https://github.com/saidsikira/Overdrive    03. [L0611] 301 https://github.com/alexdrone/Dispatch  → https://github.com/alexdrone/DispatchStore    04. [L0945] 301 https://github.com/kean/Yalta  → https://github.com/kean/Align    05. [L1291] 301 https://github.com/JustHTTP/Just  → https://github.com/dduan/Just    06. [L1620] 404 https://www.adobe.io/apis/creativecloud/creativesdk.html     07. [L1624] 404 https://www.goprimer.com     08. [L1951] 301 https://github.com/yahoo/YangMingShan  → https://github.com/YahooArchive/YangMingShan    09. [L1970]  https://github.com/dhirajjadhao/Newly Failed to open TCP connection to github.com:443 (Connection timed out - connect(2) for ""github.com"" port 443)",5776,vsouza,awesome-ios,566968587,,,,
nein,nein,false,Error: Cannot find module 'fast-deep-equal'     at Function.Module._resolveFilename (module.js:325:15)     at Function.Module._load (module.js:276:25)     at Module.require (module.js:353:17)     at require (internal/module.js:12:17)     at Object. (/home/travis/build/electron/electron/node_modules/request/node_modules/har-validator/node_modules/ajv/lib/compile/resolve.js:4:13)     at Module._compile (module.js:409:26)     at Object.Module._extensions..js (module.js:416:10)     at Module.load (module.js:343:32)     at Function.Module._load (module.js:300:12),13465,electron,electron,303195467,"no, the build failed for another reason","E: Failed to fetch http://dl.hhvm.com/ubuntu/dists/trusty/main/binary-i386/Packages  404  Not Found [IP: 13.33.164.117 80]  E: Some index files failed to download. They have been ignored, or old ones used instead.  Traceback (most recent call last):    File ""./script/cibuild"", line 129, in < module>      sys.exit(main())    File ""./script/cibuild"", line 63, in main      execute(['sudo', 'apt-get', 'update'])    File ""/home/travis/build/electron/electron/script/lib/util.py"", line 174, in execute      raise e  subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'update']' returned non-zero exit status 100",,
,ja,true,"* Failure 404 Not Found thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: NotFound, error: StringError(""https://static.redox-os.org/pkg/x86_64-unknown-redox/ipcd.sig not found"") }', libcore/result.rs:945:5",3771,redox-os,redox,387092985,,,,
,ja,true,README.md:2186:Line longer than 80 characters.,232,mhinz,vim-galore,374778890,,,,
,ja,true,README.md:2186:Line longer than 80 characters.,233,mhinz,vim-galore,374779116,,,,
,ja,true,README.md:2186:Line longer than 80 characters.,234,mhinz,vim-galore,374779422,,,,
,ja,true,README.md:2180:Line longer than 80 characters.,235,mhinz,vim-galore,374779992,,,,
,ja,true,README.md:2180:Line longer than 80 characters.,236,mhinz,vim-galore,374781325,,,,
,ja,true,README.md:2180:Line longer than 80 characters.,237,mhinz,vim-galore,374803396,,,,
,ja,true,README.md:2192:Line longer than 80 characters.,238,mhinz,vim-galore,403259414,,,,
,ja,true,"rake aborted! No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)",55557,angular,angular,476567056,,,,
,ja,true,"ERROR: AddressSanitizer: SEGV on unknown address 0x000001d548d0 (pc 0x000000455084 bp 0x000000000000 sp 0x7ffcdbc362b0 T0) = ==16577==The signal is caused by a WRITE memory access. =     #0 0x455083 in atomic_compare_exchange_strong__sanitizer::atomic_uint8_t> /tmp/final/llvm.src/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h:81:10 =     #1 0x455083 in AtomicallySetQuarantineFlagIfAllocated /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:554 =     #2 0x455083 in __asan::Allocator::Deallocate(void*, unsigned long, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_allocator.cc:631 =     #3 0x4fb97b in __interceptor_free /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_malloc_linux.cc:129:3 =     #4 0x1066974 in xfree /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:119:3 =     #5 0x12c54b8 in free_string_option /home/travis/build/neovim/neovim/build/../src/nvim/option.c:2306:5 =     #6 0x12c49dd in free_all_options /home/travis/build/neovim/neovim/build/../src/nvim/option.c:969:9 =     #7 0x1069609 in free_all_mem /home/travis/build/neovim/neovim/build/../src/nvim/memory.c:650:3",34137,neovim,neovim,567711736,,,,
,ja,true,Overfull hbox (19.64194pt too wide) in paragraph at lines 7256--7268 Overfull hbox (19.98665pt too wide) in paragraph at lines 7256--7268,5803,cplusplus,draft,566217161,,,,
,ja,true,Overfull hbox (19.64194pt too wide) in paragraph at lines 7256--7268 Overfull hbox (19.98665pt too wide) in paragraph at lines 7256--7268,5805,cplusplus,draft,566236411,,,,
,ja,true,Overfull hbox (2.26166pt too wide) in paragraph at lines 4826--4842,5808,cplusplus,draft,566259827,,,,
,ja,true,Overfull hbox (2.26166pt too wide) in paragraph at lines 4826--4842,5810,cplusplus,draft,566261609,,,,
,ja,true,Overfull hbox (2.26166pt too wide) in paragraph at lines 4826--4842,5836,cplusplus,draft,566741544,,,,
,ja,true,LaTeX Warning: There were undefined references.,6135,cplusplus,draft,568620326,,,,
,ja,true,LaTeX Warning: There were undefined references.,6136,cplusplus,draft,568621273,,,,
,ja,true,LaTeX Warning: There were undefined references.,6137,cplusplus,draft,568621492,,,,
,ja,true,"Alamofire_macOS_Tests.SessionTestCase    testThatDataRequestWithInvalidURLStringThrowsResponseHandlerError, [31mXCTAssertTrue failed[0m    [36m/Users/travis/build/Alamofire/Alamofire/Tests/SessionTests.swift:550[0m    ```  [38;5;230m        [39m[38;5;221;01mif[39;00m[38;5;230m [39m[38;5;221;01mlet[39;00m[38;5;230m [39m[38;5;153;01merror[39;00m[38;5;230m [39m[38;5;87m=[39m[38;5;230m [39m[38;5;230mresponse[39m[38;5;87m?[39m[38;5;87m.[39m[38;5;230merror[39m[38;5;87m?[39m[38;5;87m.[39m[38;5;230masAFError[39m[38;5;230m [39m[38;5;87m{[39m[38;5;230m",1832,Alamofire,Alamofire,553190487,,,,
ja,nein,false,"No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb) /home/travis/.rvm/gems/ruby-2.2.6/bin/ruby_executable_hooks:15:in `eval' /home/travis/.rvm/gems/ruby-2.2.6/bin/ruby_executable_hooks:15:in `'",2463,technomancy,leiningen,243309632,"no, the build failed for another reason","I don't use Travis any more, I switched to CircleCI. I thought that removing my Travis config would stop it from building; I had no idea it would keep going for years after I stopped using it.",,
nein,nein,false,error[E0463][0m[0m[1m: can't find crate for `fontconfig`[0m  [0m  [0m[0m[1m[38;5;12m--> [0m[0mfont/src/lib.rs:28:1[0m  [0m   [0m[0m[1m[38;5;12m|[0m  [0m[1m[38;5;12m28[0m[0m [0m[0m[1m[38;5;12m| [0m[0mextern crate fontconfig;[0m  [0m   [0m[0m[1m[38;5;12m| [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^^[0m[0m [0m[0m[1m[38;5;9mcan't find crate,3755,jwilm,alacritty,564527447,"no, the build failed for another reason",,,
nein,nein,false,[35] SSL connect error (error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure),3773,jwilm,alacritty,565672174,"no, the build failed for another reason",,,
,ja,true,[0K$ ./check_integrity.sh $(find bin | cut -b 5- | xargs) man/git-line-summary.md is required for bin/git-line-summary,310,tj,git-extras,562612733,,,,
nein,nein,false,,55253,discourse,discourse,566815121,,,,
,ja,true,"The command ""$TIMEOUT 35m ci/build.sh"" exited with 124.",8310,purescript,purescript,549434310,,,,
,ja,true,Access violation in generated code when reading ffffffffffffffff,8311,purescript,purescript,549447715,,,,
ja,nein,false,/home/travis/build/purescript/purescript/sdist-test/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build/Language/PureScript/CST/Parser.hs:60:64: error:     Not in scope: type variable 'a'    | 60 | newtype HappyWrap27 = HappyWrap27 (QualifiedName (N.ProperName a))    |                                                                ^  /home/travis/build/purescript/purescript/sdist-test/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build/Language/PureScript/CST/Parser.hs:67:55: error:     Not in scope: type variable 'a'    | 67 | newtype HappyWrap28 = HappyWrap28 (Name (N.ProperName a)),8312,purescript,purescript,549463948,"close, but not quite correct","The portion extracted is correct, but it only represents a part of the entire reason. I think the actual reason would ideally be all of lines 5065 - 5129.",,
,ja,true,"The command ""$TIMEOUT 35m ci/build.sh"" exited with 124.",8314,purescript,purescript,551305362,,,,
ja,nein,false,"/home/travis/build/purescript/purescript/sdist-test/[;1msrc/Language/PureScript/Environment.hs:528:3: [;1m[31merror:[0m[0m[;1m[0m[0m[;1m[0m [0m    • Couldn't match type 'Bool -> TypeClassData' with 'TypeClassData'[0m [0m      Expected type: M.Map[0m [0m                       (Qualified (ProperName 'ClassName)) TypeClassData[0m [0m        Actual type: M.Map[0m [0m                       (Qualified (ProperName 'ClassName)) (Bool -> TypeClassData)[0m [0m    • In the expression:[0m [0m        M.fromList[0m [0m          [(primSubName C.moduleCoerce ""Coercible"", [0m [0m            makeTypeClassData[0m",8360,purescript,purescript,573089600,"close, but not quite correct","This extracted error is mostly correct, but it omitted the following three lines, which are the snippet of code where the problem occurred:      |  528 |   M.fromList      |   ^^^^^^^^^^...",,
,ja,true,"Error found:         in module [33mMain[0m         at /home/travis/build/purescript/purescript/sdist-test/tests/purs/passing/Coercible.purs:6:1 - 6:28 (line 6, column 1 - line 6, column 28)                    Module [33mSafe.Coerce[0m was not found.           Make sure the source file exists, and that it has been provided as an input to the compiler.",8361,purescript,purescript,573124345,,,,